brev-bestilling-boot/src/main/java/no/nav/pensjon/brev/bestilling/consumer/BrevMetaData.java

```diff
@@ -0,0 +1,46 @@
+package no.nav.pensjon.brev.bestilling.consumer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public class BrevMetaData {
```

Comment:
BrevMetaData is not a spring managed component. Is the @Value working at all ?

Reply:
https://github.com/navikt/pensjon-brev-bestilling/blob/b82b0f556bc68e96aff16c0b5b80e936d8bf001e/brev-bestilling-boot/src/main/java/no/nav/pensjon/brev/bestilling/consumer/TestBrevMetaDataEndpoint.java#L12 ser ganske spring managed

Reply:
Ja en den er opprettet via
@Bean
    public BrevMetaData brevMetaData() {
        return new BrevMetaData();
    }

Jeg ville ikke forvente at @Value vil fungere med` "new BrevMetaData();"`
Kanskje det er noe jeg ikke ser ?